relay-server/src/actors/processor.rs

```diff
@@ -2139,6 +2139,10 @@ impl EnvelopeProcessorService {
                 };
             }
             Err(error) => {
+                relay_statsd::metric!(
+                    counter(RelayCounters::HandleProcessEnvelopeError) += 1,
+                    internal_error = if error.is_internal() { "true" } else { "false" }
```

Comment:
```suggestion
                    internal_error = &error.is_internal().to_string()
```

would something like this work?  This `if` just looks strange here.

Reply:
Good suggestion 👍. I misread the documentation and understood there was no `ToString` or `FromBool` implemented.